README.md

```diff
@@ -3,6 +3,9 @@
 This module allows restricting users, that match given regular expressions,
 from performing actions such as creating rooms or sending invites.
 
+Note that server administrators are not bound by these rules, as the code
+within Synapse exempts server administrators from 'spam checks'.
```

Comment:
Hmm, it looks like this isn't *entirely* accurate.

We don't exempt homeserver admins from upgrading a room: https://github.com/matrix-org/synapse/blob/0171fa5226a6aa808d9965dab20f22f9794810d9/synapse/handlers/room.py#L408-L411

Or from sending "spammy" events: https://github.com/matrix-org/synapse/blob/fef2e792beec9c126953b4f6b6d2d9f6e31ed96f/synapse/handlers/message.py#L869

So I might scope this claim to those checks that are relevant to this Synapse module (inviting another user, creating a new room).